sklearn/preprocessing/data.py

```diff
@@ -503,11 +511,10 @@ class StandardScaler(BaseEstimator, TransformerMixin):
 
     See also
     --------
-    :func:`sklearn.preprocessing.scale` to perform centering and
-    scaling without using the ``Transformer`` object oriented API
+    scale: Equivalent function without the object oriented API
```

Comment:
Same as [this comment](https://github.com/scikit-learn/scikit-learn/pull/6748/files#r61788862)